saleor/settings.py

```diff
@@ -348,30 +348,17 @@ def get_host():
 
 LOGOUT_ON_PASSWORD_CHANGE = False
 
+ENABLE_SEARCH = False
 
 ELASTICSEARCH_URL = os.environ.get('ELASTICSEARCH_URL')
-SEARCHBOX_URL = os.environ.get('SEARCHBOX_URL')
-BONSAI_URL = os.environ.get('BONSAI_URL')
-# We'll support couple of elasticsearch add-ons, but finally we'll use single
-# variable
-ES_URL = ELASTICSEARCH_URL or SEARCHBOX_URL or BONSAI_URL or ''
-if ES_URL:
-    SEARCH_BACKENDS = {
+if ELASTICSEARCH_URL:
+    ENABLE_SEARCH = True
+    INSTALLED_APPS.append('django_elasticsearch_dsl')
```

Comment:
Why cannot this library be always in installed apps?

Reply:
Application subscribes for django signals on object change for index synchronization purpose, which is good thing but excessive in case of elasticsearch not configured. I thought it would be best to turn it off entirely instead of relying on app config to turn everything off. On external library change saleor might start doing weird stuff.